### ISSUE TITLE
Nautilus breaks with /usr/share/nautilus-python/extensions/open-tilix.py

### ISSUE THREAD

**Issue by wfpaisa** [gnunn1/tilix#1594]
```
Traceback (most recent call last):
  File "/usr/share/nautilus-python/extensions/open-tilix.py", line 118, in get_file_items
    filename = file_.get_name().decode('utf-8')
AttributeError: 'str' object has no attribute 'decode'
```

**Comment by bilelmoussaoui** [comment]
how can I reproduce this, please?

**Comment by wfpaisa** [comment]
making a search in Nautilus, 

OS: Manjaro 18.0.0 Illyria
Kernel: x86_64 Linux 4.14.85-1-MANJARO
Uptime: 3d 23h 8m
Packages: 1378
Shell: zsh 5.6.2
Resolution: 1920x1200
DE: GNOME 
WM: GNOME Shell
WM Theme: Plane-dark
GTK Theme: Plane-dark [GTK2/3]
Icon Theme: PlaneDarkGnomedark
Font: Roboto 10
CPU: Intel Core i5-2410M @ 4x 2.9GHz [63.0°C]
GPU: GeForce GT 525M
RAM: 5959MiB / 7962MiB

**Comment by egmontkob** [comment]
> AttributeError: 'str' object has no attribute 'decode'

Sounds like a python 2 vs. 3 issue to me.

**Comment by bilelmoussaoui** [comment]
@egmontkob thanks for the tip. Didn't look that close to the error msg.

**Comment by chrisjbillington** [comment]
This was fixed in PR #1610. Python 2 will be dead soon, but the extension was modified to work with py2 and py3.

**Comment by timhughes** [comment]
@gnunn1 looks like this can be closed :-)
